homeassistant/components/motionmount/__init__.py

```diff
@@ -0,0 +1,48 @@
+"""The Vogel's MotionMount integration."""
+from __future__ import annotations
+
+import motionmount  # type: ignore[import-untyped]
```

Comment:
Since you're the dev behind the library, maybe add a `py.typed` file in the library to avoid these imports https://peps.python.org/pep-0561/